src/com/theironyard/novauc/TravelCost.java

```diff
@@ -0,0 +1,76 @@
+package com.theironyard.novauc;
+
+/**
+ * Created by JacobP on 2/7/17.
+ */
+public class TravelCost {
+    int distance;
+    double mpg;
+    double pricePerGallon;
+    double lodgingCost;
+    double totalCost;
+
+
+    public TravelCost(int distance, double mpg, double pricePerGallon, double lodgingCost, double totalCost) {
+        this.distance = distance;
+        this.mpg = mpg;
+        this.pricePerGallon = pricePerGallon;
+        this.lodgingCost = lodgingCost;
+        this.totalCost = totalCost;
+    }
+
+    public double getMpg() {
+        return mpg;
+    }
+
+    public void setMpg(double mpg) {
+        this.mpg = mpg;
+    }
+
+    public double getPricePerGallon() {
+        return pricePerGallon;
+    }
+
+    public void setPricePerGallon(double pricePerGallon) {
+        if (pricePerGallon < 0 && pricePerGallon > 50) {
+            this.pricePerGallon = pricePerGallon;
```

Comment:
Check the logic on line 35. I think you might have been aiming for an OR statement.  So || instead of &&.